File: Golang/1-50/8.go
package main

import (
	"fmt"
	"math"
	"os"
)

func main() {
	var a, b, c, d, e, mn, mx int64

	fmt.Fscan(os.Stdin, &a)
	fmt.Fscan(os.Stdin, &b)
	fmt.Fscan(os.Stdin, &c)
	fmt.Fscan(os.Stdin, &d)
	fmt.Fscan(os.Stdin, &e)

	mn = int64(math.Min(math.Min(float64(a), float64(b)), math.Min(float64(c), math.Min(float64(d), float64(e)))))
	mx = int64(math.Max(math.Max(float64(a), float64(b)), math.Max(float64(c), math.Max(float64(d), float64(e)))))

	fmt.Printf("%d %d\n", a+b+c+d+e-mx, a+b+c+d+e-mn)
}

File: C/1-50/8.c
#include <stdio.h>

long long min(long long a, long long b) {
    return (a < b) ? a : b;
}

long long max(long long a, long long b) {
    return (a > b) ? a : b;
}

int main() {
    long long a, b, c, d, e, mn, mx;
    scanf("%lld %lld %lld %lld %lld", &a, &b, &c, &d, &e);
    mn = min(min(a, b), min(c, min(e, d)));
    mx = max(max(a, b), max(c, max(e, d)));
    printf("%lld %lld", a + b + c + d + e - mx, a + b + c + d + e - mn);
    return 0;
}
